dev/fb/cli.sh

```diff
@@ -0,0 +1,41 @@
+if [[ -z "${VS_IMAGE+set}" ]]; then
+  echo "Error: VS_IMAGE environment variable is not set." >&2
+  exit 1
+elif [[ -z "$VS_IMAGE" ]]; then
+    echo "Error: VS_IMAGE environment variable is empty." >&2
+    exit 1
+fi
```

Comment:
Do we really need to distinguish between an unset and an empty env var? We don't treat them differently so I would suggest to just reduce the scripts to half the size with just this:
```suggestion
if [[ -z "$VS_IMAGE" ]]; then
    echo "Error: VS_IMAGE environment variable is empty." >&2
    exit 1
fi
```

Reply:
Updated in all files, thanks. 

First I thought to print the pre-built commands to the `build summary` with empty env values. So in case the user just copied the command and didn't provide the values I added this additional check. However, the final version prints the placeholders (`<path_to_everestctl>`, `<path_to_your_helmchart>`) so there is no point in keeping the checks.